### DISCUSSION TITLE
Form Builder

### DISCUSSION THREAD

**Discussion by joshgk00** [vanjarosoftware/Vanjaro.Platform#1574] [General]
Is the new integrated Form Builder only part of the hosted site option in Vanjaro or is/will the Form Builder be part of the available Blocks in the DNN Extension? Is it included in the 1.1 release?

**Comment by Tychodewaard** [comment]
I have 1.1 running and the form builder is not there. The form builder is part of the hosted version or you can buy the DNN extension Live Forms from Mandeeps.com

**Reply by joshgk00** [reply]
I thought it might be only in the hosted version. Thanks for confirming. I do have Live Forms installed on a site running 1.1 and it's not the same integrated experience as the hosted version but the module still works.

**Reply by meetmandeep** [reply]
The Vanjaro Form Builder with integrated experience will be available for purchase in Vanjaro App Store that is coming soon.
